spec/requests/api/arbitration_default_spec.rb

```diff
@@ -0,0 +1,134 @@
+RSpec.describe 'Arbitration Default API' do
+  let(:ems) { FactoryGirl.create(:ext_management_system) }
+
+  context 'arbitration defaults index' do
+    it 'can list the arbitration defaults' do
+      FactoryGirl.create(:arbitration_default, :ems_id => ems.id)
+
+      api_basic_authorize collection_action_identifier(:arbitration_defaults, :read, :get)
+      run_get(arbitration_defaults_url)
+      expect_query_result(:arbitration_defaults, 1, 1)
+    end
+  end
+
+  context 'arbitration defaults create' do
+    let(:request) do
+      {:ems_id => ems.id}
+    end
+
+    it 'rejects creation without appropriate role' do
+      api_basic_authorize
+
+      run_post(arbitration_defaults_url, request)
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'supports single arbitration_default creation' do
+      api_basic_authorize collection_action_identifier(:arbitration_defaults, :create)
```

Comment:
Very minor, but I prefer to group these lines according to the four-phase test, so a line between setup, exercise, verify and teardown (if any) steps.